doc/changes/DM-45722.other.md

```diff
@@ -0,0 +1 @@
+Explanatory logs for "initial data ID query returned no rows" are now reported at ERROR, not CRITICAL, level.
```

Comment:
This should be "misc". We decided to merge misc and other but I see we forgot to edit the README about that. Would you mind deleting the "other" entry from `doc/changes/README.rst` please?